### PR TITLE
BUG/MAINT:  TIMED SABER version and github actions

### PR DESCRIPTION
**Pull request by jklenzing**
# Description

TIMED SABER released a new version of data, and the filenames and directories have changed.  Updates to use the latest.

Also updates github actions python version to 3.

## Type of change

- Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Standard tests.  Since remote location and filename have changed, files will not download with old code.

## Test Configuration
* Operating system: Mac OS X Monterrey
* Version number: Python 3.8.11

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes
- [x] Update zenodo.json file for new code contributors

### CODE REVIEW

**Comment by jklenzing** [comment]
last pull before I can create the RC

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -21,9 +21,9 @@ jobs:
     name: Python ${{ matrix.python-version }} on ${{ matrix.os }} with numpy ${{ matrix.numpy_ver }}
     runs-on: ${{ matrix.os }}
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v2
+      uses: actions/setup-python@v3
```
Comment (aburrell) [inline .github/workflows/main.yml:26]:
```suggestion
      uses: actions/setup-python@v4
```

Found out there's a newer version.

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,11 +15,13 @@ This project adheres to [Semantic Versioning](https://semver.org/).
   * Fixed a bug in loading ICON IVM data (added multi_file_day = True)
   * Allow for array-like OMNI HRO meta data
   * Fixed date handling for OMNI HRO downloads
+  * Updated filenames for timed saber
 * Maintenance
   * Reduce duplication of code in instrument modules
   * Include flake8 linting of docstrings and style in Github Actions
   * Move OMNI HRO custom functions to a methods module
   * Deprecate OMNI HRO custom functions in instrument module
+  * Update github actions to v3
```
Comment (aburrell) [inline CHANGELOG.md:24]:
```suggestion
  * Update GitHub actions to the latest versions
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -15,11 +15,13 @@ This project adheres to [Semantic Versioning](https://semver.org/).
   * Fixed a bug in loading ICON IVM data (added multi_file_day = True)
   * Allow for array-like OMNI HRO meta data
   * Fixed date handling for OMNI HRO downloads
+  * Updated filenames for timed saber
```
Comment (aburrell) [inline CHANGELOG.md:18]:
```suggestion
  * Updated filenames for TIMED SABER
```

**Review by aburrell — CHANGES_REQUESTED** [review]

**Comment by jklenzing** [comment]
Code is updated and running successfully, but coveralls is broken (failing here and other pulls)

**Comment by jklenzing** [comment]
Coveralls working again.  Ready for review

**Review by aburrell — APPROVED** [review]

**Comment by rstoneback** [comment]
Thanks @jklenzing and @aburrell

**Pull request merged**